CraftingRevisions/CraftingRevisions.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's going on here?

Reply:
not sure, can't locate this is any of my commits